packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -327,17 +340,28 @@ export default class Editor {
         return isNodeEmpty(this.core.contentDiv, trim);
     }
 
+    public isDarkMode(): boolean {
+        return this.core.inDarkMode;
+    }
+
+    public getDarkModeOptions(): DarkModeOptions {
+        return this.core.darkModeOptions;
+    }
+
     /**
      * Get current editor content as HTML string
      * @param triggerExtractContentEvent Whether trigger ExtractContent event to all plugins
      * before return. Use this parameter to remove any temporary content added by plugins.
      * @param includeSelectionMarker Set to true if need include selection marker inside the content.
      * When restore this content, editor will set the selection to the position marked by these markers
+     * @param getInDarkMode Set to true if you want to get the content of the editor in dark mode
+     * This is a no-op when in light mode. If true, instead of converting to light, it will return the 'real' editor content.
      * @returns HTML string representing current editor content
      */
     public getContent(
         triggerExtractContentEvent: boolean = true,
-        includeSelectionMarker: boolean = false
+        includeSelectionMarker: boolean = false,
+        getInDarkMode: boolean = false,

```

Comment:
getInDarkMode is a misleading name (seems like you're getting what the content will be in darkmode, and shouldn't relate to the state of the editor body on the DOM).

I'd suggest something like `normalizeColor = true`. That way you're communicating that by default a transform on the editor's body is being done here.



Also -- all these options are triggering my code smell. This seems like a good candidate for an options kwarg function.